### PR TITLE
Add covid by race data, use it in a time series and bar chart, and clean up variable providers

### PR DESCRIPTION
**Pull request by aaronsn**

### CODE REVIEW

File: app/src/features/charts/LineChart.tsx [hunk 1]
```diff
@@ -1,21 +1,23 @@
 import React from "react";
 import { Vega } from "react-vega";
+import { Row } from "../../utils/DatasetTypes";
 
 function LineChart(props: {
-  data: {}; // for instance, race
+  data: Row[]; // for instance, race
```
Comment (kkatzen) [inline app/src/features/charts/LineChart.tsx:6]: does the "for instance, race" comment still make sense?

File: app/src/features/reports/ChartDumpReport.tsx [hunk 2]
```diff
@@ -46,6 +42,10 @@ function ChartDumpReport() {
               </ul>
             </div>
             <TimeReport />
```
Comment (kkatzen) [inline app/src/features/reports/ChartDumpReport.tsx:44]: I think we can go ahead and delete this one now

File: app/src/utils/FakeMetadataMap.ts [hunk 3]
```diff
@@ -28,7 +28,7 @@ const fakeMetadata: DatasetMetadata[] = [
       },
       {
         data_type: "string",
-        name: "race",
+        name: "hispanic_or_latino_and_race",
```
Comment (kkatzen) [inline app/src/utils/FakeMetadataMap.ts:31]: maybe rename to ethnicity_and_race ? at first glance I was confused thinking this variable referred to hispanic/latino ethnicity only

**Review by kkatzen — APPROVED** [review]

File: app/src/utils/FakeMetadataMap.ts (hunk 3 above)
Comment (aaronsn) [reply app/src/utils/FakeMetadataMap.ts:31]: Can I do this as a followup? I've littered this constant throughout the code and I'm worried about breaking things. I want to refactor commonly used column names to not be hard-coded anyway, so I can change this as part of that.

**Pull request merged**